src/catchup/CatchupWork.cpp

```diff
@@ -172,8 +174,9 @@ CatchupWork::verifyLedgers(LedgerRange const& range)
     CLOG(INFO, "History")
         << "Catchup verifying ledger chain for checkpointRange ["
         << range.first() << ".." << range.last() << "]";
-    mVerifyLedgersWork = addWork<VerifyLedgerChainWork>(
-        *mDownloadDir, range, mManualCatchup, mFirstVerified, mLastVerified);
+    mVerifyLedgersWork =
+        addWork<VerifyLedgerChainWork>(*mDownloadDir, range, mFirstVerified,
```

Comment:
we should probably use this occasion to get rid of this "pass by reference" that is quite ugly (`mFirstVerified`) and set it when `mVerifyLedgersWork` succeeds with something like `mFirstVerified  = mVerifyLedgersWork->getFirstVerified()`.
Also, maybe rename it to something a bit less cryptic - I don't know what "first verified" means especially now that we verify backwards. Maybe `mVerifiedRangeStart` ? (also rename the corresponding member in `VerifyLedgerChainWork`)